doc/http2-implementation-notes.md

```diff
@@ -50,21 +50,19 @@ const server = http2.createSecureServer(options, (req, res) => {
 
```

Comment:
Likely best to leave out the changes to this file... these implementation notes are out of date and will be replaced entirely very soon by proper documentation.